### PR TITLE
[SOLVED] Fixing typos and grammar from Codédex blog posts

### PR DESCRIPTION
**Pull request by MilanGrujicic**
This PR proposes a solution pointed out in this [issue](https://github.com/codedex-io/blog/issues/20).

### CODE REVIEW

**Comment by MilanGrujicic** [comment]
@sonnynomnom @asiqurrahman Could you please verify?

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 1]
```diff
@@ -11,34 +11,34 @@ tags:
 
 _Our "A Day in the Life" series is an inside view into what it’s like working in tech for Gen Zs. From college interns getting their first peek into tech to recent professionals who’ve been on the scene, we’ll share their experience, inspiration, tips & tricks, and more!_
 
-_In the second issue, we are featuring Ellie Popoca (she/they), our amazing Curriculum Intern this summer at Codédex. Enjoy!_
+_In the second issue, we are featuring Ellie Popoca (she/they), our amazing curriculum intern this summer at Codédex. Enjoy!_
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:14]: revert!

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 2]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
 ##### Hiii Ellie! Tell us a little about yourself.
 
-Hello! My name is Ellie, and I attend the University of Illinois at Urbana Champaign (the midwest of the U.S. and lots and lots of 🌽)! I study [Information Science](https://ischool.illinois.edu), which is the interdisciplinary study of data, human computer interaction, and web culture! I also study [Game Design](https://informatics.ischool.illinois.edu/game-studies-design) because of my interest in video games. During my free time, I like drawing, watching video essays, and rock climbing! 
+Hello! My name is Ellie, and I attend the University of Illinois at Urbana Champaign (the midwest of the U.S. and lots and lots of 🌽!) I study [information science](https://ischool.illinois.edu), which is the interdisciplinary study of data, human-computer interaction, and web culture! I also study [game design](https://informatics.ischool.illinois.edu/game-studies-design) because of my interest in video games. During my free time, I like drawing, watching video essays, and rock climbing!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:18]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 3]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
 Fun facts: The ✨ emoji is my favorite to add to just about any sentence. I’m never found without my matcha.
 
 ##### What do you do at Codédex?
 
-At Codédex, I am a part of the Curriculum team! I have written a couple of project tutorials, such as [Build a Soundboard with p5.js](https://www.codedex.io/projects/build-an-interactive-soundboard-with-p5js) and Deploy a Web Page with Vercel (coming soon!). I also help with reviewing course exercises and bonus articles that everyone sees on Codédex, plus some fun side projects that I get to collaborate with the Community and Engineering teams.
+At Codédex, I am a part of the curriculum team! I have written a couple of project tutorials, such as [Build a Soundboard with p5.js](https://www.codedex.io/projects/build-an-interactive-soundboard-with-p5js) and Deploy a Web Page with Vercel (coming soon!). I also help with reviewing course exercises and bonus articles that everyone sees on Codédex, plus some fun side projects that I get to collaborate on with the community and engineering teams.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:24]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 4]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
 ![BeReals](https://imgur.com/Q9km4Fg.png)
 <div style={{textAlign: "center"}}>(I’ve been doing BeReal’s all summer; some went off during my working hours! 😎)</div>
 
 ##### How did you end up working with us?
 
-I met Sonny through [FreyHacks](https://www.freyhacks.tech), a hackathon for which I was a [Major League Hacking](https://mlh.io) (MLH) Coach! There, I learned about Codédex, the team, and their mission! I followed Codédex for a while until I was reached out by them earlier this year to potentially be an intern, and I immediately was so excited to join when meeting the team. ✨
+I met Sonny through [FreyHacks](https://www.freyhacks.tech), a hackathon for which I was a [Major League Hacking](https://mlh.io) (MLH) coach! There, I learned about Codédex, the team, and their mission! I followed Codédex for a while until I was reached out to by them earlier this year to potentially be an intern, and I was immediately so excited to join when meeting the team. ✨
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:31]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 5]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
-##### What are the best aspects of working as a Curriculum Intern?
+##### What are the best aspects of working as a curriculum intern?
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:33]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 6]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
-Some of the best aspects of working as a Curriculum Intern here at Codédex is getting to work with all the really great people who started it! Getting to understand and review curriculum from past educators is really interesting to me, and I couldn’t have imagined getting to work with both the gaming and education side of tech.
+Some of the best aspects of working as a curriculum intern here at Codédex are getting to work with all the really great people who started it! Getting to understand and review curriculum from past educators is really interesting to me, and I couldn’t have imagined getting to work with both the gaming and education sides of tech.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:35]: revert the first one and keep the second two changes

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 7]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
-Also, I love talking to everyone in our Discord community, it’s been a fun time! 😌
+Also, I love talking to everyone in our Discord community; it’s been a fun time! 😌
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:37]: revert i think

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 8]
```diff
@@ -11,34 +11,34 @@ tags:
...
 
 ##### What does a typical day look like for you?
 
-- I start my day writing my agenda and a [standup meeting](https://en.wikipedia.org/wiki/Stand-up_meeting) with the rest of the team! 
+- I start my day by writing my agenda and having a [standup meeting](https://en.wikipedia.org/wiki/Stand-up_meeting) with the rest of the team!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:41]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 9]
```diff
@@ -48,9 +48,9 @@ Also, I love talking to everyone in our Discord community, it’s been a fun tim
 
 ##### Do you have any advice for younger students?
 
-One of my biggest tips is choosing your concentration early on! It's fun to experiment with different areas of tech that you might be interested in whether that be software engineering, web dev, data science, or even stuff like product or project management! Getting to know what you like and REALLY getting good at those skills in that particular area will help a ton when you’re trying to get your desired internship! 
+One of my biggest tips is to choose your concentration early on! It's fun to experiment with different areas of tech that you might be interested in, whether that be software engineering, web development, data science, or even stuff like product or project management! Getting to know what you like and REALLY getting good at those skills in that particular area will help a ton when you’re trying to get your desired internship!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:51]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 10]
```diff
@@ -64,17 +64,17 @@ Getting into college! I know it’s really typical, but as a first-generation co
 
 ##### What’s your dream job? (besides Codédex ;)
 
-BESIDES Codédex, my dream job would be to be a PM ([Product Manager](https://en.wikipedia.org/wiki/Product_manager)) for Duolingo or Electronic Arts! Working on a game would be a super awesome experience getting to work with incredibly talented people and engineers. 
+Besides Codédex, my dream job would be to be a PM ([product manager](https://en.wikipedia.org/wiki/Product_manager)) for Duolingo or Electronic Arts! Working on a game would be a super awesome experience, getting to work with incredibly talented people and engineers.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:67]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 11]
```diff
@@ -64,17 +64,17 @@ Getting into college! I know it’s really typical, but as a first-generation co
...
 
-Completely unrelated, something about a dream job in a more unrealistic sense is getting to work with innovating over with the folks at [The Browser Company](https://thebrowser.company) making Arc Browser! I’d love to explore the different ways in which people use and interact with browsers, and what potential it could reach! Something about the upbringing of the Internet Computer is incredibly inspiring me and gets me excited to work on tech in the next 10 years. 🚀
+Completely unrelated, something about a dream job in a more unrealistic sense is getting to work on innovating over with the folks at [The Browser Company](https://thebrowser.company) making Arc Browser! I’d love to explore the different ways in which people use and interact with browsers and what potential they could reach! Something about the upbringing of the Internet computer is incredibly inspiring and gets me excited to work on tech in the next 10 years. 🚀
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:69]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 12]
```diff
@@ -64,17 +64,17 @@ Getting into college! I know it’s really typical, but as a first-generation co
...
 
 ![team selfie pic](https://imgur.com/fIVd4sa.png)
 
-##### What inspired you to get into Computer Science?
+##### What inspired you to get into computer science?
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:73]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 13]
```diff
@@ -64,17 +64,17 @@ Getting into college! I know it’s really typical, but as a first-generation co
...
 
-Since I was little, I always loved technology and cool internet design and of course, video games! Growing up, I always wanted to be some sort of engineer. It was my biggest passion for so long. In high school, I did robotics throughout my entire time there, and it wasn’t until 12th grade that I discovered [hackathons](https://en.wikipedia.org/wiki/Hackathon)! I didn’t know how to code websites or anything, but I was super excited to learn! Luckily, I joined a team of other beginners, and we ended up winning best visual design! 
+Since I was little, I have always loved technology, and cool internet design, and, of course, video games! Growing up, I always wanted to be some sort of engineer. It was my biggest passion for so long. In high school, I did robotics throughout my entire time there, and it wasn’t until 12th grade that I discovered [hackathons](https://en.wikipedia.org/wiki/Hackathon)! I didn’t know how to code websites or anything, but I was super excited to learn! Luckily, I joined a team of other beginners, and we ended up winning best visual design!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:75]: revert

File: blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx [hunk 14]
```diff
@@ -64,17 +64,17 @@ Getting into college! I know it’s really typical, but as a first-generation co
...
+Since I was little, I have always loved technology, and cool internet design, and, of course, video games! Growing up, I always wanted to be some sort of engineer. It was my biggest passion for so long. In high school, I did robotics throughout my entire time there, and it wasn’t until 12th grade that I discovered [hackathons](https://en.wikipedia.org/wiki/Hackathon)! I didn’t know how to code websites or anything, but I was super excited to learn! Luckily, I joined a team of other beginners, and we ended up winning best visual design! 
 
-Being validated at that time made me feel like, despite being a beginner, I could still contribute to the tech space. Ever since then, I attended more digital hackathons, and learned more and more about web design! Starting college was a great way for me to also meet like-minded people that shared the same passions as me! 
+Being validated at that time made me feel like, despite being a beginner, I could still contribute to the tech space. Ever since then, I have attended more digital hackathons and learned more and more about web design! Starting college was a great way for me to also meet like-minded people who shared the same passions as me!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-curriculum-intern-codedex-ellie-popoca.mdx:77]: revert

File: blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx [hunk 15]
```diff
@@ -15,21 +15,21 @@ _In the third issue, we are featuring Miracle Awonuga (she/her), a recent gradua
 
 ##### Hiii Miracle! Tell us a bit about yourself.
 
-Hello everyone! My name is Miracle Awonuga. I was born and raised in Antioch, TN, a city 30 minutes southeast of Nashville. I am a proud first generation Nigerian-American. I recently graduated from [Belmont University](https://belmont.edu) in Nashville with a data science degree. I am the first Black woman to achieve this in the university’s history. 
+Hello everyone! My name is Miracle Awonuga. I was born and raised in Antioch, TN, a city 30 minutes southeast of Nashville. I am a proud first-generation Nigerian-American. I recently graduated from [Belmont University](https://belmont.edu) in Nashville with a data science degree. I am the first Black woman to achieve this in the university’s history.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx:18]: revert

File: blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx [hunk 16]
```diff
@@ -15,21 +15,21 @@ _In the third issue, we are featuring Miracle Awonuga (she/her), a recent gradua
...
+Hello everyone! My name is Miracle Awonuga. I was born and raised in Antioch, TN, a city 30 minutes southeast of Nashville. I am a proud first-generation Nigerian-American. I recently graduated from [Belmont University](https://belmont.edu) in Nashville with a data science degree. I am the first Black woman to achieve this in the university’s history. 
 
 I am also the former co-Director of People and Communications at [WaffleHacks](https://wafflehacks.org), where I fostered relationships with our sponsors, partners, and community members. 
 
 In my free time, I like to play piano, do nail art, and watch video essays!
 
 ##### What are you currently up to? 
 
-I am getting ready to start grad school! I will be working towards my Masters in Data Science at [Middle Tennessee State University](https://www.mtsu.edu) located in Murfreesboro, TN. 
+I am getting ready to start grad school! I will be working towards my Masters in Data Science at [Middle Tennessee State University](https://www.mtsu.edu), located in Murfreesboro, TN.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx:26]: revert

File: blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx [hunk 17]
```diff
@@ -15,21 +15,21 @@ _In the third issue, we are featuring Miracle Awonuga (she/her), a recent gradua
...
 
 ##### What are you currently up to? 
 
-I am getting ready to start grad school! I will be working towards my Masters in Data Science at [Middle Tennessee State University](https://www.mtsu.edu) located in Murfreesboro, TN. 
+I am getting ready to start grad school! I will be working towards my Masters in Data Science at [Middle Tennessee State University](https://www.mtsu.edu), located in Murfreesboro, TN. 
 
 ##### How did you get into the field?
 
-Fun fact: I started college as a computer science major! I realized the spring of my freshman year that CS was not for me while struggling through an exam for my Programming I class. I felt like computer science limited my ability to pursue all of my passions and be creative. I heard a voice whisper inside of me begging me to consider switching to data science. 
+Fun fact: I started college as a computer science major! I realized in the spring of my freshman year that CS was not for me while struggling through an exam for my Programming I class. I felt like computer science limited my ability to pursue all of my passions and be creative. I heard a whisper inside of me begging me to consider switching to data science.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx:30]: revert

File: blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx [hunk 18]
```diff
@@ -15,21 +15,21 @@ _In the third issue, we are featuring Miracle Awonuga (she/her), a recent gradua
...
+Fun fact: I started college as a computer science major! I realized in the spring of my freshman year that CS was not for me while struggling through an exam for my Programming I class. I felt like computer science limited my ability to pursue all of my passions and be creative. I heard a whisper inside of me begging me to consider switching to data science. 
 
-After doing research and checking out what Belmont had to offer with the field, I knew I had to switch my major. Making the switch to data science is one of the top 5 best decisions I’ve made because I have more resources and skills to pursue my passions in ways I never thought were possible.
+After doing research and checking out what Belmont had to offer in the field, I knew I had to switch my major. Making the switch to data science is one of the top 5 best decisions I’ve made because I have more resources and skills to pursue my passions in ways I never thought were possible.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-data-science-student-miracle-awonuga.mdx:32]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 19]
```diff
@@ -11,23 +11,23 @@ tags:
...
+_In the first issue, we are featuring Brandon Lam (he/him), a SWE intern at the NASA Glenn Research Center, ISS, and Human Research Office. Enjoy!_
 
 ##### Hellooo Brandon! Tell us a bit about yourself.
 
-Hey, I’m Brandon! I’m a Software Engineering Intern at the National Aeronautics and Space Administration (NASA) and an incoming junior at the University of Minnesota studying Computer Science. In my free time, I play strategy games like [Teamfight Tactics](https://en.wikipedia.org/wiki/Teamfight_Tactics) (TFT) and [Hearthstone](https://en.wikipedia.org/wiki/Hearthstone). I met the Codédex team at HackNYU and loved the platform. Nice to meet you guys!
+Hey, I’m Brandon! I’m a software engineering intern at the National Aeronautics and Space Administration (NASA) and an incoming junior at the University of Minnesota studying computer science. In my free time, I play strategy games like [Teamfight Tactics](https://en.wikipedia.org/wiki/Teamfight_Tactics) (TFT) and [Hearthstone](https://en.wikipedia.org/wiki/Hearthstone). I met the Codédex team at HackNYU and loved the platform. Nice to meet you guys!
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:18]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 20]
```diff
@@ -11,23 +11,23 @@ tags:
...
 
-Fun fact: I’ve peaked top 100 in North America in both games. 🙂
+Fun fact: I’ve peaked in the top 100 in North America in both games. 🙂
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:20]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 21]
```diff
@@ -11,23 +11,23 @@ tags:
...
 
 ##### What do you do at NASA?
 
-I’m a SWE Intern at [NASA Glenn Research Center](https://en.wikipedia.org/wiki/Glenn_Research_Center) in Cleveland, OH, updating and testing an inventory management site this summer. The internship is two months — from June 5th to August 11th. I’m mainly doing fullstack web development using [Spring Boot](https://spring.io/projects/spring-boot), [JavaScript](https://www.codedex.io/javascript), [Node.js](https://nodejs.org), and [MySQL](https://www.mysql.com).
+I’m a SWE intern at [NASA Glenn Research Center](https://en.wikipedia.org/wiki/Glenn_Research_Center) in Cleveland, OH, updating and testing an inventory management site this summer. The internship is two months—from June 5th to August 11th. I’m mainly doing full-stack web development using [Spring Boot](https://spring.io/projects/spring-boot), [JavaScript](https://www.codedex.io/javascript), [Node.js](https://nodejs.org), and [MySQL](https://www.mysql.com).
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:24]: revert

Comment (sonnynomnom) [reply blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:24]: keep capitalized intern and fullstack > full-stack i think. more pro

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 22]
```diff
@@ -11,23 +11,23 @@ tags:
...
 
 ![Brandon in front of the NASA Glenn Research Center Lewis Field](https://imgur.com/s8hm0mU.png)
 
 ##### How did you get the internship?
 
-I applied last December on [intern.nasa.gov](intern.nasa.gov) and got a a call in March from my current mentor, Nancy. She was interested in my background and thought that I would be a good fit. She also mentioned she was judging for a NASA high school competition in my area and wondered if I’d like to help judge the event the next day. It was an opportunity to talk more about the position.
+I applied last December on [intern.nasa.gov](intern.nasa.gov) and got a call in March from my current mentor, Nancy. She was interested in my background and thought that I would be a good fit. She also mentioned she was judging for a NASA high school competition in my area and wondered if I’d like to help judge the event the next day. It was an opportunity to talk more about the position.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:30]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 23]
```diff
@@ -38,13 +38,13 @@ Definitely the incredible people I get to work with on a daily basis. The divers
 In addition, you can get lots of tours at the facility. Some of my favorite places have been:
 
 - Exercise Countermeasure Lab: Astronauts lose 1% bone mass every month in space, so NASA has special workout equipment created to ensure they stay in shape while on the [ISS](https://en.wikipedia.org/wiki/International_Space_Station). Astronauts in space have to work out for 4 hours each day!
-- My roommate’s chemistry lab: The microscope they use is ~20 years old, so the only way for them to retrieve data from it is to use a computer running [Windows 98](https://en.wikipedia.org/wiki/Windows_98). When I asked why they haven’t upgraded, they said, “It still works, why upgrade?” 
+- My roommate’s chemistry lab: The microscope they use is ~20 years old, so the only way for them to retrieve data from it is to use a computer running [Windows 98](https://en.wikipedia.org/wiki/Windows_98). When I asked why they haven’t upgraded, they said, “It still works; why upgrade?”
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:41]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 24]
```diff
@@ -38,13 +38,13 @@ Definitely the incredible people I get to work with on a daily basis. The divers
...
+- My roommate’s chemistry lab: The microscope they use is ~20 years old, so the only way for them to retrieve data from it is to use a computer running [Windows 98](https://en.wikipedia.org/wiki/Windows_98). When I asked why they haven’t upgraded, they said, “It still works; why upgrade?” 
 
 ##### What does a typical work day look like?
 
 My typical work day is no different than any other typical SWE. I get to the office every morning and check my emails. After that, I check to see if I have any meetings for the day and then get to work. Work consists of coding, attending meetings, demoing the website, and writing documentation.
 
-There're also lots of workshops offered onsite that I try to attend and my mentor really encourages networking and attending these since that’s really what the internship is about.
+There're also lots of workshops offered onsite that I try to attend, and my mentor really encourages networking and attending these since that’s really what the internship is about.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:47]: revert

File: blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx [hunk 25]
```diff
@@ -63,15 +63,15 @@ If you’re interested in attending a hackathon, sign up for the [Space Apps Cha
 
 ##### What’s a personal project that you're proud of?
 
-The personal project that I’m the proudest of would have to be my TFT Simulator. In [TFT](https://en.wikipedia.org/wiki/Teamfight_Tactics), there are lots of luck and chance involved. At the time, there was a debate that was rather 50/50 on the correct decision in the game. So I decided to solve it by using code.
+The personal project of which I’m the proudest would have to be my TFT simulator. In [TFT](https://en.wikipedia.org/wiki/Teamfight_Tactics), there are lots of luck and chance involved. At the time, there was a debate that was rather 50/50 on the correct decision in the game. So I decided to solve it by using code.
```
Comment (sonnynomnom) [inline blogs/a-day-in-the-life-swe-intern-nasa-brandon-lam.mdx:66]: revert

**Comment by sonnynomnom** [comment]
ended up only taking in 5 things from this big list:

- 1 update - https://github.com/codedex-io/blog/commit/58670cf4ebf24369ab7e94698ae10e5c2e7bd7c1
- 2 updates - https://github.com/codedex-io/blog/commit/d34336a4cd9b0f0fe66a017c3c9648f9f330ebfe
- 2 updates - https://github.com/codedex-io/blog/commit/3badf253c11138ce44dc856d13c11ff18ab2a10a

**Comment by MilanGrujicic** [comment]
Jesus, I could make 10 points out of this issue. 
At least we fixed 5 typos in the blog posts.

**Pull request closed without merging**